packages/vue-renderer/src/loading.html

```diff
@@ -0,0 +1,280 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, height=device-height, initial-scale=1.0, minimum-scale=1.0">
+  <title>Nuxt.js: Loading app...</title>
+  <link href="https://fonts.googleapis.com/css?family=Quicksand:400,500" rel="stylesheet">
+  <script src="https://cdn.jsdelivr.net/npm/vue"></script>
```

Comment:
Would it be possible to just include external dependencies? It doesnt feel correct to me that in `dev` mode you are required to have an internet connection to make something like a loading screen appear as intended. I know that for 98+% of the people this would probably be ok anyway, but I am a strong believer that you should be able to develop offline without strange artifacts. Would be a perfect case for https://github.com/nuxt/rfcs/issues/22 and embedded resources ;)

Reply:
@pimlie You are right. We thought about it as well with @Atinux. The problem is that serving from `node_modules` requires extra complexities. Let's see if we can find a way for this PR or not. But finally, it should be served from local FS.